apps/distribusjon/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/distribusjon/DistribusjonRiverTest.kt

```diff
@@ -45,77 +45,50 @@ class DistribusjonRiverTest :
             clearAllMocks()
         }
 
-        test("distribuerer inntektsmelding på kafka topic") {
```

Comment:
Så noen forenklinger i de to testene som gjorde at testkroppen ble seende like ut. Flyttet dem derfor sammen under `withData`. Den eneste andre endringen er å legge til en nøkkel i `forventetRecord`.